### PR TITLE
James

### PR DESCRIPTION
**Pull request by JamesO1231**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,9 +26,10 @@
     "express": "^4.17.1",
     "express-handlebars": "^5.3.2",
     "express-session": "^1.17.2",
-    "faker": "^5.5.3",
     "jest": "^27.0.6",
     "mysql2": "^2.2.5",
-    "sequelize": "^6.6.5"
+    "node-fetch": "^2.6.1",
+    "sequelize": "^6.6.5",
+    "yup": "^0.32.9"
```
Comment (azwethinkweizkd) [inline package.json:33]: delete this

File: public/js/login.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
-const homeBtn = document.querySelector('.homeBtn');
 const registerBtn = document.querySelector('.registerBtn');
 
 const loginFormHandler = async (event) => {
     event.preventDefault();
 
+    console.log("help me")
```
Comment (azwethinkweizkd) [inline public/js/login.js:6]: delete this

File: public/js/login.js [hunk 3]
```diff
@@ -1,10 +1,11 @@
...
     const email = document.getElementById('emailLogin').value.trim();
+    console.log("Hello");
```
Comment (azwethinkweizkd) [inline public/js/login.js:8]: delete this

File: public/js/login.js [hunk 4]
```diff
@@ -15,6 +16,8 @@ const loginFormHandler = async (event) => {
             headers: { 'Content-Type': 'application/json' },
         });
 
+        console.log(response);
```
Comment (azwethinkweizkd) [inline public/js/login.js:19]: delete this

File: public/js/login.js [hunk 5]
```diff
@@ -32,8 +35,3 @@ registerBtn.addEventListener('click', function (event) {
     event.preventDefault();
     document.location.replace('/register');
 });
-
-
-document
```
Comment (azwethinkweizkd) [inline public/js/login.js:37]: keep this

File: public/js/register.js [hunk 6]
```diff
@@ -1,6 +1,3 @@
-const homeBtn = document.querySelector('.homeBtn')
```
Comment (azwethinkweizkd) [inline public/js/register.js:1]: keep this

**Review by azwethinkweizkd — DISMISSED** [review]

**Review by bonilhav — APPROVED** [review]

**Pull request merged**